📝 Add documentation for media server (#42)
ajnart/homarr-docs: docs/widgets/media-server.mdx

---

> title: 🎞️ Media Server

Reviewer: ```suggestion
title: 🎬 Media Server
```

tags:

>   - Widgets
>   - Integration
>   - Configuration
>   - Media Server
>   - Plex
>   - Jellyfin
>   - Sessions
>   - API

Reviewer: ```suggestion
  - Widgets
  - Integration
  - Configuration
```

Keep to the required tags.

Author: I disagree. Why would we remove those?
They help with indexing and the search. If you just enter Jellyfin, you'll find this page.

Reviewer: They don't help with indexing and search. They are purely for categorizing on the tags page. Our search plugin can already search contents of the page.

Author: Okay, fair. I will apply your suggestion.

---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

The media server widget enables you to watch and view current active sessions over all your media servers.
It also displays useful information, that you can use when providing support to users or debugging a problem.

---

## Adding the widget
Please check out our documentation on [how to add a widget](/docs/introduction/after-the-installation#adding-widgets).

> ---

Reviewer: Please add some form of "Configuring" section here. Just like all other widget docs.

Author: Why? There is no configuration for this widget

Reviewer: All widget docs have a configuration section, even if there aren't really things to configure.

> ## Common pitfalls

Reviewer: ```suggestion
## Common issues
```

Is the widget not working? Please try out the following tips and tricks to possibly resolve your issue:

<Tabs>
  <TabItem value="jellyfin" label="Jellyfin" default>

> - Use a URL that does not terminate on a signle ``/`` or something like ``/web`` or ``index.html``. This may break the integration. Only use the basename of your server
> - Ensure that your container has access to the media server. It's recommended that you make a ping or telnet
> - Use valid credentials, that have been created for Homarr in the Jellyfin administration dashboard.
> - Permit access to the required libraries and grant the required permissions

Reviewer: ```suggestion
- Use a URL that does not terminate on a single ``/`` or something like ``/web`` or ``index.html``. This may break the integration. Only use the main domain of your server.
- Make sure that your container has access to the media server. Try making a ping to it.
- Use valid credentials, that have been created for Homarr in the Jellyfin administration dashboard.
- Permit access to the required libraries and grant the required permissions.
```

  </TabItem>
  <TabItem value="plex" label="Plex">

> - Use a URL that does not terminate on a signle ``/`` or something like ``/web`` or ``index.html``. This may break the integration. Only use the basename of your server
> - Ensure that your container has access to the media server. It's recommended that you make a ping or telnet
> - Use a token, that has been created from your Plex. You can retrieve the token using this official guide: https://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/
>   - Use a token that has not expired
>   - No token visible? Try logging out and in again
> - Do not use the cloud or remote URLs. The integration will only work for local instances. Do not worry, as Homarr will act as a primitve reverse proxy when you're outside the network of Plex
> - Permit access to the required libraries and grant the required permissions

Reviewer: ```suggestion
- Use a URL that does not terminate on a single ``/`` or something like ``/web`` or ``index.html``. This may break the integration. Only use the main domain of your server.
- Make sure that your container has access to the media server. Try making a ping to it.
- Use a token, that has been created from your Plex. You can retrieve the token using this official guide: https://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/
  - Make sure the token isn't expired.
  - No token visible? Try logging out and in again.
- Do not use the cloud or remote URLs. The integration will only work for local instances. Do not worry, as Homarr will act as a primitve reverse proxy when you're outside the network of Plex.
- Permit access to the required libraries and grant the required permissions.
```
